iobasa/Harry-Harry
# Dev.ife.product reviews

### Goal
Add review app to store

### Deployment Instructions
-Pull master
-Ensure that your production id is correct in `config.yml`
- `theme deploy --env production`


### Testing Instructions
- [ ] go to the [ Product detail page PDP](https://co4wqtmo4ao1sp0m-31237603464.shopifypreview.com)
-scroll down and write review
-refresh the page and ensure that the review is showing
-scroll to the top of the page ensures that badge is showing near the price

- [ ] go to  [Collection page](https://ife-fanshawe.myshopify.com/collections/all)
-ensure that the collection
- ensure the products with no reviews have a review badge

### Tested On
Chrome on Windows

### Code Request Quality Checklist


### Reviewers
@reviewer1  
@reviewer2  

### Mentions
@mention1  
@mention2
